cluster-service/deploy/templates/cloud-resource-constraints-config.configmap.yaml

```diff
@@ -1,20 +1,280 @@
 apiVersion: v1
-kind: ConfigMap
-metadata:
-  name: cloud-resource-constraints-config
-  namespace: '{{ .Release.Namespace  }}'
 data:
-  instance-type-constraints.yaml: |
-    instance_types:
-      - id: Standard_D8s_v3
-        ccs_only: true
-        enabled: true
-      - id: Standard_D8ps_v6
-        ccs_only: true
-        enabled: true
   cloud-region-constraints.yaml: |
     cloud_regions:
       - id: '{{ .Values.region  }}'
         enabled: true
         govcloud: false
         ccs_only: false
+  instance-type-constraints.yaml: |
+    instance_types:
+    # hand-crafted list to the ones mentioned in https://issues.redhat.com/browse/XCMSTRAT-1002
+    # those are the instance types that are also enabled in ARO Classic
```

Comment:
Have we verified that we can create a node pool (up to it being ready) using the added instances types?

Reply:
I'll do some more testing today, but I was able to create a new node pool yesterday with one of those

Reply:
adding one of every category using your demo script, e.g.:
> $ NP_NAME=test-nc6s ./04-create-nodepool.sh

![image](https://github.com/user-attachments/assets/b01ce014-23e9-45d0-b61d-fa4e8173575e)

The GPU instances for example don't have quota, they are available in the region for selection but we can't provision them:
![image](https://github.com/user-attachments/assets/1028428c-679a-4adf-82e8-354c70d4978e)

Let me know if we want to disable them in the meantime, I'll let Jerome know in the ticket.

Reply:
Thank you for running the verification steps. The lack of quota is perhaps just on our subscription and likely this won't be observed in other subscription (e.g customer's ones). I am okay with having them enabled.
However, we should note that we've not been able to verify them and once the quota are there, we should verify them is what I'd suggest.